src/mesh/mechanical/solid/femMesh.cpp

```diff
@@ -122,9 +122,7 @@ void femMesh::allocate_displacement_boundary(Json::Value const& boundary, BasicM
             return dof + dof_offset;
         });
 
-        displacement_bcs[boundary_name].emplace_back(boundary_dofs,
-                                                     boundary["Time"],
-                                                     boundary["Values"][name]);
+        displacement_bcs[boundary_name].emplace_back(boundary_dofs, boundary, std::string(name));
```

Comment:
Tight.  See above, I'm not sure if there would be a compiler error without `std::string(x)`